doc/classes/FileAccess.xml

```diff
@@ -474,7 +474,7 @@
 			<return type="void" />
 			<param index="0" name="line" type="String" />
 			<description>
-				Appends [param line] to the file followed by a line return character ([code]\n[/code]), encoding the text as UTF-8.
+				Stores [param line] in the file followed by a line return character ([code]\n[/code]), encoding the text as UTF-8.
```

Comment:
I think we should fix the "line return" wording here as I'd say it's confusing, there's two characters:
* `\n` newline character 
* `\r` carriage return character

So:
```suggestion
				Stores [param line] in the file followed by a newline character ([code]\n[/code]), encoding the text as UTF-8.
```